tests/integration/models/gpflux/test_predictions.py

```diff
@@ -65,31 +75,37 @@ def test_dgp_model_close_to_simple_implementation(depth: int, keras_float: None)
 
     example_data = hartmann_6_dataset(dataset_size)
 
+    # optimization settings
```

Comment:
feels slightly annoying that we need to redefine an optimizer every time..

Reply:
yes, perhaps we should redo the default optimizer settings

Reply:
It should be possible to create a utility function that does this for us?